test/test-project/test-registry.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment node
+ */
+
+const { readFile } = require('fs').promises;
+const path = require('path');
+const Generator = require('../lib/generator');
+const dummySpecPath = path.resolve(__dirname, './docs/dummy.yml');
+const crypto = require('crypto');
+const mainTestResultPath = 'test/temp/integrationTestResult';
+//we do not want to download chromium for html-template if it is not needed
+process.env['PUPPETEER_SKIP_CHROMIUM_DOWNLOAD'] = true;
```

Comment:
is it really possible to set envs like that in runtime? I think better move it to docker compose

Reply:
Removed @derberg

Reply:
not removed